CPPLINT.cfg

```diff
@@ -0,0 +1,71 @@
+#
+# Cpplint is a command-line tool to check C/C++ files for style issues following Google's C++ style guide.
+# https://github.com/cpplint/cpplint
+#
+# Type the command below for details about supported options in CPPLINT.cfg.
+# > cpplint --help
+#
+
+# Do not look for additional CPPLINT.cfg in the parent directories.
+set noparent
+root=.
+extensions=cpp,hpp,c,h
+
+# Limit line length.
+linelength=120
+
+# Adjust the following error categories as specified by the filter:
+# (filter parameters are concatenated together)
+
+# `build` rules
+# Enable a warning about C++ features that were not in the original
+# C++11 specification (and so might not be well-supported).  In the
+# case of F´, the minimum supported platforms are potentially not
+# new enough to afford to do without this warning.
+filter=+build/c++11
+
+# F´ uses `#ifndef FOO_H` guard not the `#pragma once`.
+filter=+build/header_guard
```

Comment:
This `+build/header_guard` line should also be removed, some warnings still make it through.